test/uk/gov/hmrc/gform/core/TemplateValidatorSpec.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.gform.core
 
```

Comment:
could rename this class to FormTemplateValidatorSpec, as its unit tests for FormTemplateValidator

Reply:
Out of the scope of this ticket.

Reply:
No worries